app/jobs/application_job.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class ApplicationJob < ActiveJob::Base
```

Comment:
In a new `ApplicationJob` file, usually, the following two comments appear. We can check whether we want to enable some of this behavior for our use case:

```rb
  # Automatically retry jobs that encountered a deadlock
  # retry_on ActiveRecord::Deadlocked

  # Most jobs are safe to ignore if the underlying records are no longer available
  # discard_on ActiveJob::DeserializationError
```

Reply:
I think deadlocks should not be possible right now and DeserializationErrors should not occur because we now pass the uuid instead of the entire task to the SyncJob. The SyncJob then handles both existing tasks and not existing tasks.

Reply:
Okay, that's true. Still (you know me 🤓), I would suggest to copy these comments to our file (and would potentially enable them, since they don't harm and are als applicable to other jobs [future jobs and integrated jobs, such as mail sendings, which will also be handled by SolidQueue now, or ActiveStorage purge jobs]).

Reply:
Done